src/commands/new.ts

```diff
@@ -7,13 +7,15 @@ import { resolve } from 'path';
 
 const { writeFile, readFile } = fPromises;
 const DEFAULT_ASYNCAPI_FILE_NAME = 'asyncapi.yaml';
+const DEFAULT_ASYNCAPI_TEMPLATE = 'basic.yaml';
 
 export default class New extends Command {
   static description = 'creates a new asyncapi file';
 
   static flags = {
     help: flags.help({ char: 'h' }),
     'file-name': flags.string({ char: 'n', description: 'name of the file' }),
+    'selected-template': flags.string({ char: 'n', description: 'name of the template to use' }),
```

Comment:
A few things on this:

1. Shall we call it `template` or `example`? I think we're not giving the user a template they can fill in the gaps but an example they can modify.
2. I'd just go for `example` (or `template`, if we prefer). It's implicit that you're selecting it because you're passing the flag.
3. The char can't be `n` as it's already being used for the `file-name` flag. Maybe we can go for `e` (example) or `t` (template).

So here's my suggestion you can apply if you like it:

```suggestion
    'example': flags.string({ char: 'e', description: 'name of the example to use' }),
```

Reply:
Yeah good point @fmvilas , yeah will make the change to move to `example` a few places to change so will make them seperate. Thanks